lazy-imports-import.html

```diff
@@ -38,17 +38,17 @@
       var links =
         Polymer.DomModule.import(fromElement.localName).querySelectorAll(query);
 
-        if (links.length < 1) {
-          return Promise.resolve(importStatuses);
-        }
+      if (links.length < 1) {
+        return Promise.reject(importStatuses);
```

Comment:
Reject should be called with an error though. The error could carry the statuses I guess.